content/ngf/install/openshift.md

```diff
@@ -0,0 +1,220 @@
+
+  1. Navigate to the Red Hat Catalog: https://catalog.redhat.com/en  
+  2. Search for "NGINX Gateway Fabric Operator" in the searchbar at the top
+  3. Select NGINX Gateway Fabric Operator
+  4. Select **Deploy & use**. 
+  5. Choose the appropriate architecture and release tag
+  6. Complete the installation. Wait until the Operator status shows Installed
+
+### Create a project
+
+In your cluster, create a dedicated project (namespace) for NGINX Gateway Fabric components.
+
+```shell
+oc new-project nginx-gateway-fabric
+```
+
+### Create TLS secrets for internal communication (optional)
+
+If you want NGINX Gateway Fabric to auto-generate internal certificates, skip this step. To provide your own TLS secrets, create the following:
+
+Agent TLS (used by internal agent)
+
+```shell
+oc create secret tls agent-tls \
+  --cert=agent.crt \
+  --key=agent.key \
+  -n nginx-gateway-fabric
+```
+
+Server TLS (used by internal server)
+
+```shell
+oc create secret tls server-tls \
+  --cert=server.crt \
+  --key=server.key \
+  -n nginx-gateway-fabric
+```
+
+### Integrate with NGINX One Console (optional)
+
+If you want to use NGINX One Console to monitor NGINX Gateway Fabric, create a secret for the dataplane key (replace VALUE with your key).
+
+```shell
+oc create secret generic nginxone-dataplane-key \
+  --from-literal=key=VALUE \
+  -n nginx-gateway-fabric
+```
+
+Reference this secret in `spec.nginx.nginxOneConsole.dataplaneKeySecretName`.
+
+### Configure NGINX Plus licensing (optional)
+
+If you plan to use NGINX Plus, set `spec.nginx.plus: true`, add image pull credentials, and create a license secret if needed.
+
+Example license secret name referenced by `usage.secretName`
+
+```shell
+oc create secret generic nplus-license \
+  --from-file=nginx-repo.crt=/path/to/nginx-repo.crt \
+  --from-file=nginx-repo.key=/path/to/nginx-repo.key \
+  -n nginx-gateway-fabric
+```
+
+### Create the NginxGatewayFabric custom resource
+
+Create a minimal `NginxGatewayFabric` custom resource for OpenShift. Include this code in a file named `nginx-gateway-fabric.yaml`.
+
+```yaml
+   apiVersion: gateway.nginx.org/v1alpha1
+   kind: NginxGatewayFabric
+   metadata:
+     name: ngf
+     namespace: nginx-gateway-fabric
+   spec:
+     # Data plane (NGINX)
+     nginx:
+       replicas: 2
+       image:
+         repository: ghcr.io/nginx/nginx-gateway-fabric/nginx
+         tag: 2.2.0-ubi
+         pullPolicy: IfNotPresent
+
+     # Controller
+     nginxGateway:
+       gatewayClassName: nginx
+       gatewayControllerName: gateway.nginx.org/nginx-gateway-controller
+       image:
+         repository: ghcr.io/nginx/nginx-gateway-fabric
+         tag: 2.2.0-ubi
+         pullPolicy: IfNotPresent
+       replicas: 1
+   ```
+
+Apply the custom resource:
+
+```shell
+oc apply -f nginx-gateway-fabric.yaml
+```
+
+Result: The Operator reconciles the custom resource and provisions the NGINX Gateway Fabric controller and data plane.
```

Comment:
This is incorrect - the data plane will only be provisioned when a Gateway object has been created